bolt-sidecar/src/signer/commit_boost.rs

```diff
@@ -34,14 +30,19 @@ pub enum CommitBoostError {
     #[error("failed to sign constraint: {0}")]
     NoSignature(String),
     #[error("failed to create signer client: {0}")]
-    SignerClientError(#[from] ErrReport),
+    SignerClientError(#[from] SignerClientError),
+    #[error("error in commit boost signer: {0}")]
+    Other(#[from] eyre::Report),
 }
 
+type Result<T> = std::result::Result<T, CommitBoostError>;
```

Comment:
in general I'm not a big fan of renaming std library objects, it's confusing. I understand you did this to avoid replacing all instances of the word, but I'm sure this is just 1 vim macro away :)

I would simply rename this to `CBResult` or `CommitBoostResult`. then you also don't need to qualify std::result::Result anymore.

Reply:
I have seen this comment only now, I wasn't 100% happy with the current errors either and I've made a commit that should resolve your comment as well: [`ddaedb5` (#287)](https://github.com/chainbound/bolt/pull/287/commits/ddaedb52c89dce8604a0d36570dc9ac21d85c9ea). wdyt?

Reply:
yep awesome, ty